src/lib/y2partitioner/widgets/columns.rb

```diff
@@ -29,7 +29,6 @@ module Columns
 require "y2partitioner/widgets/columns/chunk_size"
 require "y2partitioner/widgets/columns/device"
 require "y2partitioner/widgets/columns/encrypted"
-require "y2partitioner/widgets/columns/filesystem_id"
```

Comment:
I think `filesystem_devices` must also be removed. It's pretty much the same case than `filesystem_id`.